src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.targets

```diff
@@ -26,6 +26,10 @@
     -->
     <_GetChildProjectCopyToPublishDirectoryItems>false</_GetChildProjectCopyToPublishDirectoryItems>
     <UseMonoRuntime Condition=" '$(UseMonoRuntime)' == '' ">true</UseMonoRuntime>
+
+    <!-- Mono components -->
+    <AndroidEnableProfiler Condition=" '$(AndroidEnableProfiler)' == ''">false</AndroidEnableProfiler>
```

Comment:
The `$(AndroidIncludeDebugSymbols)`  initialization should also be updated to take `$(AndroidEnableProfiler)` into consideration: https://github.com/xamarin/xamarin-android/blob/7e9f595f23ebd5b53385b16e3890bc90f95f6fef/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L302-L323

Reply:
What do you mean @jonpryor? The profiler is independent on whether or not the build is a debug one, it can be used in both after all